Gulpfile.js

```diff
@@ -53,6 +60,97 @@ function rename(fn) {
   });
 }
 
+/**
+ *
+ * @typedef {("asserts" | "builders" | "constants" | | "validators")} HelperKind
+ * @param {HelperKind} helperKind
+ */
+function generateTypeHelpers(helperKind) {
+  const dest = `./packages/babel-types/src/${helperKind}/generated/`;
+  const formatCode = require("./scripts/utils/formatCode");
+  return gulp
+    .src(".", { base: __dirname })
+    .pipe(errorsLogger())
+    .pipe(
+      through.obj(function (file, enc, callback) {
+        file.path = "index.js";
+        file.contents = Buffer.from(
+          formatCode(
+            require(`./packages/babel-types/scripts/generators/${helperKind}`)(),
+            dest + file.path
+          )
+        );
+        fancyLog(`${chalk.green("✔")} Generated ${helperKind}`);
+        callback(null, file);
+      })
+    )
+    .pipe(gulp.dest(dest));
+}
+
+function buildFlowTypings() {
+  const dest = "./packages/babel-types/lib/";
+  return gulp
+    .src(".", { base: __dirname })
+    .pipe(errorsLogger())
+    .pipe(
+      through.obj(require("./packages/babel-types/scripts/generators/flow.js"))
+    )
+    .pipe(rename(() => "index.js.flow"))
+    .pipe(gulp.dest(dest));
+}
+
+function buildTypeScriptTypings() {
+  const dest = "./packages/babel-types/lib/";
+  return gulp
+    .src(".", { base: __dirname })
+    .pipe(errorsLogger())
+    .pipe(
+      through.obj(
+        require("./packages/babel-types/scripts/generators/typescript.js")
+      )
+    )
+    .pipe(rename(() => "index.d.ts"))
+    .pipe(gulp.dest(dest));
+}
+
+function generateStandalone() {
+  const dest = "./packages/babel-standalone/src/generated/";
+  const formatCode = require("./scripts/utils/formatCode");
+  return gulp
+    .src(babelStandalonePluginConfigGlob, { base: __dirname })
+    .pipe(
+      through.obj((file, enc, callback) => {
+        fancyLog("Generating @babel/standalone files");
+        const pluginConfig = JSON.parse(file.contents);
+        let imports = "";
+        let list = "";
+        let allList = "";
+
+        for (const plugin of pluginConfig) {
+          const camelPlugin = camelCase(plugin);
+          imports += `import ${camelPlugin} from "@babel/plugin-${plugin}";`;
+          list += `${camelPlugin},`;
+          allList += `"${plugin}": ${camelPlugin},`;
+        }
+
+        const fileContents = `/*
+ * This file is auto-generated! Do not modify it directly.
+ * To re-generate run 'yarn gulp generate-standalone'
+ */
+${imports}
+
+export {${list}};
+
+export const all = {${allList}};
```

Comment:
Another nit: Since we are calling `formatCode` on this anyway, we can probably indent this template.